app/views/devise/sessions/new.html.erb

```diff
@@ -2,6 +2,9 @@
 	<h2>Sign In</h2>
 </div>
 <%= simple_form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <% flash.each do |key, value| %>
```

Comment:
@abnersajr What do you think about making it more reusable?
```ruby
# app/helpers/application_helper.rb

module ApplicationHelper
  def flash_class_for(flash_type)
    { 
      success: "alert-success", 
      error: "alert-danger", 
      alert: "alert-warning", 
      notice: "alert-info" 
    }[flash_type.to_sym] || flash_type.to_s
  end
end
```

```erb
# app/views/shared/_flash.html.erb

<% flash.each do |key, message| %>
  <div class="alert <%= flash_class_for(key) %>" role="alert">
    <%= message %>
  </div>
<% end %>
```

```erb
# app/views/devise/sessions/new.html.erb

<%= simple_form_for(...) do |f| %>
  <%= render 'shared/flash' %>
  # ...
```

Reply:
Diggin into the code I found another helper like this one.